platforms/melpomene/melpo.toml

```diff
@@ -0,0 +1,44 @@
+[kernel_cfg]
+max_drivers = 16
+timer_granularity = { secs = 0, nanos = 1000 } # 1us
+
+[kernel_svc_cfg]
+keyboard_mux = { max_keyboards = 8, buffer_capacity = 32, sermux_port = 2 }
+serial_mux = { max_ports = 16, max_frame = 512 }
+spawnulator = { capacity = 16 }
+sermux_loopback = { port = 0, buffer_size = 128 }
+
+[kernel_svc_cfg.sermux_hello]
+port = 1
+buffer_size = 32
+message = [ # this sucks
```

Comment:
oh yeah we might want to do something about this, maybe switch to `heapless::String<32>` instead of `[u8; 32]`. Open to ideas here.